web/src/routes/(user)/search/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -259,6 +259,8 @@
             {#await getPersonName(value) then personName}
               {personName}
             {/await}
+          {:else if value === null || value === ''}
```

Comment:
```suggestion
          {:else if !value}
```

Reply:
That might work right now, it's not actually correct. It would make searching for a numeric value of 0 or `false` show the wrong value. `!value` impies "falsy", but what we actually want is "null or empty string".

Reply:
That's fair, it could lead to wrong behavior down the line.